### ISSUE TITLE
Tool Input Recently Broken

### ISSUE THREAD

**Issue by zacfield-stream** [Vanhal/ProgressiveAutomation#167]
In version 1.6.24, there were some changes made that affect the way items are piped into the machines. Using the chopper as an example, previously you could have a pipe connected that is pulling tools out of an inventory into the side of a chopper, and it would only pull a tool when the previous one broke. Since the update, pipes/hoppers will continue to push tools into the chopper (being placed in the output slots) as they don't recognize that the tool slot is the only valid input. Possible solutions would be to either return item handling as it was prior to 1.6.24, or add a feature on the PA Wrench for ToolInput.

**Comment by HowardRoark2016** [comment]
I am playing FtB Horizons: Daybreaker v1.02 which by default uses ProgressiveAutomation-1.7.10-1.6.16. That is the version of Progressive Automation I was using until just a moment ago when I tried to input wheat to an Animal Breeder and realized that nothing (not a vanilla hopper attached to any side of the Breeder, not an Extra Utilities transfer pipe, not a Mekanism's advanced logistical transporter pipe) would insert the wheat into the Breeder. Up until now, I had already built a fully self-contained Progressive Automation tree farm and wheat farm. What I mean is, and this is the point of this post, that I was using Mekanism's logistical transport pipes attached to the bottom of Progressive Automation's Crafter because that seemed to be the only way to ensure that only the crafted item would be pulled out and not either the fuel or crafting ingredients. Also, I had used both Mekanism's logistical transport pipes attached to the sides of Crafters to pump recipe items in, and I had used Extra Utilities transfer pipes attached to the sides of Breeders to pump tools in. Until now, everything pumped into Progressive Automation machines seemed to know where it was meant to go.

Then I got to the point in the build where I was going to pump wheat into the Breeders (to close the circle on the fully self-contained animal breeding farm). Nothing I tried was successful in pumping items into the Breeder. I looked to the Github to see if there was an update for Progressive Automation. I saw ProgressiveAutomation-1.7.10-1.6.24 was recently released so I downloaded it and installed it. Now, my Progressive Automation machines are not filling up as before. The machines no longer seem to know where items are supposed to go. When I pump a hoe into a Planter, the first hoe goes into the tool slot, but now the pipes no longer think the machine is full, so more hoes are pumped in and fill the Planter's inventory slots. The same thing happened to the Chopper. Prior to updating, I had an Extra Utilities transfer pipe pumping axes into the side of the Chopper. The transfer pipe would stop pumping the moment there was an axe in the Chopper's tool slot. Now after updating, the Chopper's tool slot fills with one axe but then continues to fill its inventory slots with axes. Also, I still do not see a way to get wheat to pump into the "tool" slot of the Breeder. Wheat pumped into the Breeder seems to only go into the inventory slot.
